widgetssdk/src/main/java/com/glia/widgets/messagecenter/MessageCenterController.kt

```diff
@@ -0,0 +1,27 @@
+package com.glia.widgets.messagecenter
+
+class MessageCenterController : MessageCenterContract.Controller {
+    private var view: MessageCenterContract.View? = null
+
+    override fun setView(view: MessageCenterContract.View) {
+        this.view = view
+    }
+
+    override fun onCheckMessagesClicked() {
+        view?.navigateToChat()
+    }
+
+    override fun onSendMessageClicked() {
+        view?.navigateToChat()
+    }
```

Comment:
According to Figma design, it should navigate to chat, indeed.
However, App Bar title should be 'Messaging' (not 'Chat')

Reply:
fixed